pkg/daemon/writer.go

```diff
@@ -97,6 +97,7 @@ func (nw *NodeWriter) SetUnreconcilable(err error, client corev1.NodeInterface,
 	glog.Errorf("Marking Unreconcilable due to: %v", err)
 	annos := map[string]string{
 		constants.MachineConfigDaemonStateAnnotationKey: constants.MachineConfigDaemonStateUnreconcilable,
+		constants.MachineConfigDaemonReasonAnnotationKey: err.Error(),
```

Comment:
`setDone` should also clear this annotation right?

Reply:
yup good point

Reply:
wondering if setting the annotation to an empty string actually clears it out

Reply:
fixed